projects/demo/src/modules/experimental/block-details/examples/2/index.html

```diff
@@ -0,0 +1,37 @@
+<div>
```

Comment:
Is this additional nesting is really required ?

```html
<div>
    <div tuiBlockDetails>...</div>
</div>

<div>
    <div tuiBlockDetails>...</div>
</div>
```

I see that host component has `display: flex` with `flex-direction: column`.
Could we simplify example to the following version ?

```html
<div tuiBlockDetails>...</div>

<div tuiBlockDetails>...</div>
```